### DISCUSSION TITLE
how to run   kafka-ui  locally

### DISCUSSION THREAD

**Discussion by ramamanoj142000** [provectus/kafka-ui#1719] [Q&A]
i'm  trying to build and run kafka-ui locally but it is not connecting to kafka cluster 

./mvnw spring-boot:run -Pprod -Dspring.config.location=file:///Users/m_645600/github/kafka-ui/application.yaml  

cat  /Users/m_645600/github/kafka-ui/application.yaml  --

kafka:
  clusters:
    - name: local
      bootstrapServers: isds-kafkaservice-dev-west2-lkf01.g.cognizant.com:9094
      properties:
        security.protocol: SASL_SSL
        sasl.mechanism: SCRAM-SHA-512        
        sasl.jaas.config: org.apache.kafka.common.security.scram.ScramLoginModule required username="kafka_admin" password="xxxxxxxxxxxx";

Please let us know what i'm missing .

**Comment by Haarolean** [comment]
Hi, please share your application logs.
